lib/rubocop/cop/style/keyword_arguments_merging.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # When passing an existing hash as keyword arguments, add additional arguments
+      # directly rather than using `merge`.
+      #
+      # @example
+      #   # bad
+      #   some_method(**opts.merge(foo: true))
+      #   some_method(**opts.merge(other_opts))
+      #
+      #   # good
+      #   some_method(**opts, foo: true)
+      #   some_method(**opts, **other_opts)
+      #
+      class KeywordArgumentsMerging < Base
+        extend AutoCorrector
+
+        MSG = 'Add additional arguments directly rather than using `merge`.'
```

Comment:
I'd probably say "List/Provide" instead of "Add" in this context.

Reply:
Changed.